clusters/app.ci/supplemental-ci-images/hypershift/README.md

```diff
@@ -0,0 +1,8 @@
+# Special notes for building HyperShift images
+* `hypershift-cli` image serves both as the CLI and the operator.
+* Due to lack of forward/backward-compatibility of HyperShift, it is critical that we tag the build. For instance, once a build is done and pushed to `latest`, we shall tag it like
+```
+oc --context app.ci -n ci tag hypershift-cli:latest hypershift-cli:20230428-12e6a502bd6a7ea5434df2b83fef102b7819b413 --as system:admin
+```
+The naming convention is `YYYYMMDD-<git commit hash>`.
+* If an aarch64 build is needed, we shall ensure it is built form the SAME source code as amd64, on the SAME commit
```

Comment:
```suggestion
If an image for `aarch64` is needed, we shall ensure it is built form the SAME source code as amd64, on the SAME commit.
```

Reply:
How do we archive it? I think you can remove the comment for `aarch64`. It might need bigger design to fix in our ops.

Reply:
That line was added when I tried to deploy aarch64 version of the operator to our `multi01` cluster. Image tags are shared between multiple architectures, thus we need to ensure both architectures are on the same commit to avoid confusion.